docs/source/modin/supported/series_dt_supported.rst

```diff
@@ -98,7 +98,9 @@ the method in the left column.
 +-----------------------------+---------------------------------+----------------------------------------------------+
 | ``normalize``               | Y                               |                                                    |
 +-----------------------------+---------------------------------+----------------------------------------------------+
-| ``strftime``                | N                               |                                                    |
+| ``strftime``                | P                               | ``N`` if `date_format` contains directives other   |
```

Comment:
just curios, how many directives are supported in pandas?

Reply:
This should be the full list: https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes